src/containers/User/UserArea.tsx

```diff
@@ -1,47 +1,55 @@
 import React from 'react'
-import { Button, Grid, Label, Segment, Sticky } from 'semantic-ui-react'
+import { Button, Grid, Label, Message, Segment, Sticky } from 'semantic-ui-react'
 import strings from '../../utils/constants'
 import { useUserState } from '../../contexts/UserState'
 import UserSelection from './UserSelection'
 import { useRouter } from '../../utils/hooks/useRouter'
+import useListTemplates from '../../utils/hooks/useListTemplates'
+import { AppMenu } from '../../components'
 
 const UserArea: React.FC = () => {
   const {
-    userState: { currentUser },
+    userState: { currentUser, isLoading, templatePermissions },
     logout,
   } = useUserState()
   const { replace } = useRouter()
+  const { error, loading, filteredTemplates } = useListTemplates(templatePermissions, isLoading)
 
   const handleLogOut = async () => {
     await logout()
     replace('/login')
   }
```

Comment:
Problem with the way you've re-structured the routing: When you click "Logout", the Login component is rendered _inside_ the other stuff, so the head and footer are still visible on the page. After refreshing the './login' page it appears correctly (because it hits the /login route in AppWrapper first, presumably)

Reply:
In this case (logging out), it's probably acceptable to use a javascript over-ride to force a full refresh:
`window.location.href = "/login"` (Tested it, it works)

Reply:
That's for this suggestion. I have added it now, it still has a problem, but I have added a comment so we can go back to improve this when we are working on the tyding up of User Are in Epic#10 #78

Reply:
Great. I don't even think it needs improving, I think when logging out its acceptable, even desirable, to force a refresh like this as it resets any React state that might be hanging around and does a proper "reset".

Maybe make the comment say `// Force full application reload on log out` or something like that? 

Anyway all good, approved. :)